### ISSUE TITLE
I need some help for the latest plugin 3.23.1  creat x-ray

### ISSUE THREAD

**Issue by NeuralBiophysics** [xxyzz/WordDumb#41]
### Describe the bug

I downloaded the latest version of the plugin, the dictionary function is available, but x-ray is not available for all books, the previous version can be used. I'm a new programmer and even this passage was written in a translator, so I couldn't solve the problem in the error message myself, so please give me some help if you can.

### System Information

OS: Windows 10 64
Calibre  6.1
Python 3.10
Plugin 3.23.1

### Error message

```text
calibre, version 6.1.0 (win32, embedded-python: True)
Tonnerre de Brest!: An error occurred, please copy error message then report bug at GitHub.

Starting job: Generating Word Wise and X-Ray for Harry Potter and the Philosopher's Stone 
Job: "Generating Word Wise and X-Ray for Harry Potter and the Philosopher's Stone" failed with error: 
Traceback (most recent call last):
  File "calibre\gui2\threaded_jobs.py", line 82, in start_work
  File "calibre_plugins.worddumb.parse_job", line 149, in do_job
  File "calibre_plugins.worddumb.parse_job", line 246, in create_files
  File "calibre_plugins.worddumb.parse_job", line 570, in load_spacy
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\spacy\__init__.py", line 6, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\spacy\errors.py", line 2, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\spacy\compat.py", line 3, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\thinc\__init__.py", line 5, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\thinc\config.py", line 14, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\srsly\__init__.py", line 1, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\srsly\_json_api.py", line 6, in <module>
  File "C:\Users\86153\AppData\Roaming\calibre\plugins\worddumb-libs-py3.10\srsly\ujson\__init__.py", line 1, in <module>
ModuleNotFoundError: No module named 'srsly.ujson.ujson'
 
Called with args: ((10, 'AZW3', "C:\\Users\\86153\\书库\\J.K. Rowling\\Harry Potter and the Philosopher's S (10)\\Harry Potter and the Philosophe - J.K. Rowling.azw3", <calibre.ebooks.metadata.book.base.Metadata object at 0x000001D33DCBA260>, {'spacy': 'en_core_web_', 'wiki': 'en', 'kaikki': 'English'}), True, True) {'notifications': <queue.Queue object at 0x000001D33DCBBF10>, 'abort': <threading.Event object at 0x000001D33DCB93F0>, 'log': <calibre.utils.logging.GUILog object at 0x000001D33DCB8F40>}
```

### Reproduce steps

1.chose one book
2.just click the pink star
3.no more than 10second here is this error message

**Comment by xxyzz** [comment]
Maybe something went wrong when the `srsly` package was installed. You could delete the `srsly*` folder in the error message then try again.

**Comment by NeuralBiophysics** [comment]
> 
First of all, thank you very much for your timely answer, I deleted the file under the path, re-downloaded the plug-in, the problem has been solved, thank you very much for the plug-in. There is also the ability to add customization of x-ray results. Most of the book's character annotations result in direct use of the content of the following text, most of the time it is not the character's annotation, but the spell without the original annotation directly using Wikipedia annotation is more accurate.
